atomic_reactor/yum_util.py

```diff
@@ -54,7 +54,12 @@ def filename(self):
         '''
         urlpath = unquote(urlsplit(self.repourl, allow_fragments=False).path)
         basename = os.path.basename(urlpath)
-        suffix = '-' + md5(self.repourl.encode('utf-8')).hexdigest()[:5]
+        poss_name = self.repourl[:-11] + REPO_SUFFIX
+        # check if this name is already encoded
+        if md5(poss_name.encode('utf-8')).hexdigest()[:5] == self.repourl[-10:-5]:
```

Comment:
Do we know what's causing it to be encoded twice?

Reply:
another approach